extensions/adobe/experience/offer-management/filter-rule.schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/filter-rule",
+  "meta:extensible": "true",
+  "type": "object",
+  "title": "Filter Rule",
+  "description": "Filters are used to specify crirea for offer selection",
+  "definitions": {
+    "filter-rule": {
+      "properties": {
+        "@id": {
+          "type": "string",
+          "format": "uri",
+          "title": "ID",
+          "description": "The unique identifier of the filter"
+        },
+        "xdm:name": {
+          "type": "string",
+          "title": "Name",
+          "description": "Offer name"
+        },
+        "xdm:type": {
+          "type": "string",
+          "title": "Type",
+          "description": "Filter expression type: tag rule or profile query",
+          "enum": ["tag", "profile"],
+          "meta:enum": {
+            "tag": "Tag",
+            "profile": "Profile"
+          }
+        },
+        "xdm:value": {
+          "type": "string",
+          "title": "Expression",
+          "description": "Expression: tag rule or profile query"
```

Comment:
Expression in what format?

Reply:
Depending on the selector above this is either a predicate on tags or a PQL query.

Reply:
Then please add that information to the description.  Also, links to the specifications for each.

Reply:
The file has been broken up to two: filter-expression.schema.json and eligibility-rule.schema.json . Both are well-documented now.